zebra-chain/src/block/arbitrary.rs

```diff
@@ -67,7 +89,7 @@ impl Arbitrary for Header {
                     merkle_root: merkle_root_hash,
                     root_bytes,
                     time: Utc.timestamp(timestamp, 0),
-                    difficulty_threshold,
+                    difficulty_threshold: CompactDifficulty(545259519),
```

Comment:
?

Reply:
ahahaha, one sec let me find where I calculated this

Reply:
it's this but evaluated so it doesn't have to export the constants, will be replaced once we can construct a CompactDifficulty from an Expanded difficulty

https://github.com/ZcashFoundation/zebra/blob/main/zebra-chain/src/work/difficulty/tests.rs#L148